viz_scripts/scaffolding.py

```diff
@@ -106,7 +108,7 @@ def expand_userinputs(labeled_ct):
 unique_users = lambda df: len(df.user_id.unique()) if "user_id" in df.columns else 0
 trip_label_count = lambda s, df: len(df[s].dropna()) if s in df.columns else 0
 
-def load_viz_notebook_data(year, month, program, study_type, dic_re, dic_pur=None, include_test_users=False):
+def load_viz_notebook_data(year, month, program, study_type, dynamic_labels, dic_re, dic_pur=None, include_test_users=False):
```

Comment:
I don't want to hold this up since it is not wrong, but I do think that this can be cleaned up/simplified further. It seems redundant to pass in `dynamic_labels` and `dic_re`, but not passing it in will lead to duplicated code in the notebooks. When we make this consistent with the phone mode mapping in the future, we should see if we can simplify this.

Reply:
@iantei please mark this with a 👍 or 👀 so I can see that you have read and understood it